.github/workflows/pr-verify.yml

```diff
@@ -0,0 +1,16 @@
+name: PR verify
+
+on:
+  pull_request_target:
+    types: [opened, edited, synchronize, reopened]
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+    name: verify PR contents
+    steps:
+    - name: Verifier action
+      id: verifier
+      uses: kubernetes-sigs/kubebuilder-release-tools@3c3411345eedc489d1022288aa844691e92a9c29 # tag=v0.4.2
```

Comment:
If we are going to require the use of emojis then we need to update the PR template:

https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/.github/PULL_REQUEST_TEMPLATE.md